lib/analysis.js

```diff
@@ -207,17 +208,39 @@ function getExternalRepoInfo(data) {
     };
 }
 
+function initNodeInfo(kind, typeInfo) {
+    var nodeKind = ""
+    if (kind !== undefined) {
```

Comment:
is there a chance that kind may be `false`, `null`, `0`, or empty string? if no, you can use `if (kind)`